bedrock/mozorg/templates/mozorg/contribute/tasks/joy-of-coding.html

```diff
@@ -6,26 +6,28 @@
 
 {% block page_title %}{{ _('Watch: The Joy of Coding') }}{% endblock %}
 
-{% block taskview_title %}{{ _('Watch The Joy of Coding, then discuss it with other Mozillians!') }}{% endblock %}
-{% block taskview_lead %}{{ _('Anyone can contribute to Firefox by finding and fixing errors that people report via Bugzilla. Watch a demonstration of a Firefox engineer live hacking to see what it’s all about!') }}{% endblock %}
+{% block taskview_title %}{{ _('Watch The Joy of Coding, then discuss it with other Mozillians') }}{% endblock %}
+{% block taskview_lead %}{{ _('Anyone can contribute to Firefox by finding and fixing errors that people report via Bugzilla. Watch a demonstration of a Firefox engineer live hacking to see what it’s all about.') }}{% endblock %}
 
 {% block task_content %}
   <ol class="task-steps">
-    <li id="step_one" class="one">
-      <header>
-        <h3>{{ _('Watch The Intro!') }}</h3>
+    <li>
+      <header id="step_one" class="one">
+        <h3>{{ _('Watch the intro.') }}</h3>
         <p>{{ _('<a href="https://twitter.com/mike_conley" rel="external">@mike_conley</a> livehacks on real Firefox bugs while thinking out loud.') }}</p>
       </header>
       <div class="step-content">
-        <video id="joc" preload="none" controls poster="{{ static('img/contribute/signup/joc_poster.png') }}" data-task="joyofcoding" data-step="one" data-complete="true" data-variation="1">
+        <video id="joc" preload="none" controls poster="{{ static('img/contribute/signup/joc_poster.png') }}" data-action="play" data-task="joyofcoding" data-step="one" data-complete="true">
           <source src="https://videos.cdn.mozilla.net/uploads/Videos%20creative/joc-video.mp4" type="video/mp4" />
         </video>
-        <button type="button" class="button-blue watch" data-action="play" data-task="joyofcoding" data-step="one" data-complete="true" data-variation="1">Watch Video</button>
+        <button type="button" class="button-blue watch" data-action="play" data-task="joyofcoding" data-step="one" data-complete="true">Watch Video</button>
```

Comment:
String is not wrapped. If we could use `Watch the video` it would be great, since we already have this string.